datasources/src/main/resources/datasources-index-mapping.yml

```diff
@@ -4,9 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 ##
 
-# Schema file for the observability index
-# Since we only search based on "access", sort on lastUpdatedTimeMs & createdTimeMs,
-# other fields are not used in mapping to avoid index on those fields.
+# Schema file for the .ql-datasources index
```

Comment:
can you please also update schema for datasource inside the Observability catalog ? (or create an issue for updating it ? )
 - [datasource schema](https://github.com/opensearch-project/observability/blob/main/src/main/resources/schema/system/datasource.schema)
 - [datasource sample](https://github.com/opensearch-project/observability/blob/main/docs/schema/system/samples/datasource.json)